cpp/cmake/Modules/ConfigureArrow.cmake

```diff
@@ -1,94 +1,55 @@
-set(ARROW_ROOT ${CMAKE_BINARY_DIR}/arrow)
-
-set(ARROW_CMAKE_ARGS " -DARROW_WITH_LZ4=OFF"
-                     " -DARROW_WITH_ZSTD=OFF"
-                     " -DARROW_WITH_BROTLI=OFF"
-                     " -DARROW_WITH_SNAPPY=OFF"
-                     " -DARROW_WITH_ZLIB=OFF"
-                     " -DARROW_BUILD_STATIC=ON"
-                     " -DARROW_BUILD_SHARED=OFF"
-                     " -DARROW_BOOST_USE_SHARED=ON"
-                     " -DARROW_BUILD_TESTS=OFF"
-                     " -DARROW_TEST_MEMCHECK=OFF"
-                     " -DARROW_BUILD_BENCHMARKS=OFF"
-                     " -DARROW_IPC=ON"
-                     " -DARROW_COMPUTE=OFF"
-                     " -DARROW_GPU=OFF"
-                     " -DARROW_JEMALLOC=OFF"
-                     " -DARROW_BOOST_VENDORED=OFF"
-                     " -DARROW_PYTHON=OFF"
-                     " -DARROW_TENSORFLOW=ON") # enable old ABI for C/C++
-
-configure_file("${CMAKE_SOURCE_DIR}/cmake/Templates/Arrow.CMakeLists.txt.cmake"
-               "${ARROW_ROOT}/CMakeLists.txt")
-
-file(MAKE_DIRECTORY "${ARROW_ROOT}/build")
-file(MAKE_DIRECTORY "${ARROW_ROOT}/install")
+#=============================================================================
+# Copyright 2018 BlazingDB, Inc.
+#     Copyright 2018 Percy Camilo Triveño Aucahuasi <percy@blazingdb.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#=============================================================================
+
+if (NOT ARROW_VERSION)
+    set(ARROW_VERSION "apache-arrow-0.11.1")
```

Comment:
The default version is _not_ `0.11.xx`

We fully support version `0.10.0`

Reply:
All right!